app/helpers/measures_helper.rb

```diff
@@ -1,12 +1,41 @@
 module MeasuresHelper
-  def order_number_info(record)
-    record.definition.try(:description) || "Order number #{record.number}"
+  def show_warning?(order_number)
+    order_number.definition.description && order_number.number.start_with?('05')
```

Comment:
Are we sure the `order_number.definition` and `order_number.number` will never be nil?

Reply:
Yeah, good point I think we need to guard against those cases.

Reply:
No no, we do in the view already so are already being defensive

Reply:
It's not possible for the number to be nil